Jenkinsfile

```diff
@@ -0,0 +1,203 @@
+        )
+        string(
+            name: 'FISSILE_DOCKER_REGISTRY',
+            defaultValue: '',
+            description: 'Docker registry to publish to',
+        )
+        string(
+            name: 'FISSILE_DOCKER_ORGANIZATION',
+            defaultValue: 'splatform',
+            description: 'Docker organization to publish to',
+        )
+    }
+
+    environment {
+        FISSILE_DOCKER_REGISTRY = "${params.FISSILE_DOCKER_REGISTRY}"
+        FISSILE_DOCKER_ORGANIZATION = "${params.FISSILE_DOCKER_ORGANIZATION}"
+    }
+
+    stages {
+        stage('reset') {
+            when {
+                expression { return params.RESET }
+            }
+            steps {
+                checkout(
+                    $class: scm.$class,
+                    userRemoteConfigs: scm.userRemoteConfigs,
+                    branches: scm.branches,
+                    doGenerateSubmoduleConfigurations: scm.doGenerateSubmoduleConfigurations,
+                    submoduleCfg: scm.submoduleCfg,
+                    browser: scm.browser,
+                    gitTool: scm.gitTool,
+                    extensions: [
+                        [$class: 'CheckoutOption', timeout: 30],
+                        [$class: 'CloneOption', timeout: 30],
+                        [$class: 'SubmoduleOption', recursiveSubmodules: true, timeout: 60],
+                        [$class: 'CleanBeforeCheckout'],
+                    ],
+
+                )
+            }
+        }
+        stage('clean') {
+            when {
+                expression { return params.CLEAN }
+            }
+            steps {
+                sh '''
+                    docker images --format="{{.Repository}}:{{.Tag}}" | \
+                        grep -E '/scf-|/uaa-|^uaa-role-packages:|^scf-role-packages:' | \
+                        xargs --no-run-if-empty docker rmi
+                '''
+            }
+        }
+        stage('checkout') {
+            when {
+                expression { return ! params.SKIP_CHECKOUT }
+            }
+            steps {
+                sh '''
+                    git config --global --replace-all submodule.fetchJobs 0
+                '''
+                checkout scm
+            }
+        }
+        stage('tools') {
+            steps {
+                ansiColor('xterm') {
+                    sh '''
+                        set -e +x
+                        source ${PWD}/.envrc
+                        set -x
+                        unset HCF_PACKAGE_COMPILATION_CACHE
+                        make ${FISSILE_BINARY}
+                    '''
+                }
+            }
+        }
+        stage('build') {
+            steps {
+                ansiColor('xterm') {
+                    sh '''
+                        set -e +x
+                        source ${PWD}/.envrc
+                        set -x
+                        unset HCF_PACKAGE_COMPILATION_CACHE
+                        make vagrant-prep validate
```

Comment:
No, because `validate` requires `bosh create release`.  Of course, I'm pretty sure it's actually run as part of `make vagrant-prep` already…